source/_res/rn/useful-links.txt

```diff
@@ -0,0 +1,11 @@
+Useful links
+========================
+
+- The `ProxySQL and proxysql-admin installation instructions <https://www.percona.com/doc/percona-xtradb-cluster/LATEST/howtos/proxysql-v2.html#installing-proxysql-v2>`__
+
+- The `Percona Software downloads <https://www.percona.com/downloads/>`__
+
+- The `ProxySQL and proxysql-admin GitHub location <https://github.com/percona/proxysql-admin-tool-doc>`__
```

Comment:
This is pointing to the Docs repository, instead of the actual [proxysql-admin-tool](https://github.com/percona/proxysql-admin-tool).

Reply:
Thank you! Fixed.